tests/ctst/world/Zenko.ts

```diff
@@ -52,44 +83,41 @@ export default class Zenko extends World {
 
     private forceFailed = false;
 
+    readonly parameters: Record<string, unknown> = {};
+
     private cliMode: cliModeObject = CacheHelper.createCliModeObject();
 
     /**
      * @constructor
      * @param {Object} options - parameters provided as a CLI parameter when running the tests
      */
-    constructor(options: any) {
+    constructor(options: IWorldOptions) {
         super(options);
 
         // store service users credentials from world parameters
         if (this.parameters.ServiceUsersCredentials) {
-             Object.entries(JSON.parse(this.parameters.ServiceUsersCredentials))
-                .forEach((entry : [string, any]) => {
-                    Zenko.serviceUsersCredentials[entry[0] as string] = {
-                        AccessKeyId: entry[1].accessKey as string,
-                        SecretAccessKey: entry[1].secretKey as string,
-
+            Object.entries(JSON.parse(
+                this.parameters.ServiceUsersCredentials as string) as UserCredentials)
+                .forEach(entry => {
+                    Zenko.serviceUsersCredentials[entry[0]] = {
+                        AccessKeyId: (entry[1] as { accessKey: string }).accessKey,
+                        SecretAccessKey: (entry[1] as { secretKey: string }).secretKey,
```

Comment:
This looks strange, the object is force-casted into a `UserCredentials` object, which is an object, then we iterate over it, and access its values as an array...

```suggestion
                this.parameters.ServiceUsersCredentials as string) as Record<string, UserCredentials>)
                .forEach(([key, value]) => {
                    Zenko.serviceUsersCredentials[key] = {
                        AccessKeyId: value.accessKey,
                        SecretAccessKey: value.secretKey,
```

(not tested but to illustrate the idea)

Reply:
https://github.com/scality/Zenko/pull/1818/commits/6653676e3a7b0f31994bfe1c863ca2a9293c8236#diff-371f6719b5883f728d9d3218b4ed2eaf1b7ada8faf798acf95b686a7c347c7ddL99